docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,150 @@
-# 🔧 What is the .NET Platform?
+# 🔧 What is the .NET Core Platform?
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+.NET Core is a modular, cross-platform, open source implementation of the .NET Framework that runs on Windows devices, as well as on Linux and Mac OSX. Unlike the .NET Framework, .NET Core consists of a set of runtime and library components that can be used in various configurations, depending on the needs of your application.
+
+# .NET Core and the .NET Framework #
+To get a better sense of what .NET Core is, let’s compare it to the.NET Framework 4.x:
+<table>
+<col style="width:50%">
+<col style="width:50%">
+<thead>
+   <tr>
+      <th>.NET Core</th>
+      <th>.NET Framework 4.x</th>
+   </tr>
+</thead>
+<tr>
+<td>Includes CoreCLR, a more lightweight runtime that provides basic services to your app, particularly automatic memory management and garbage collection, along with a basic type library.</td>
+<td>Includes the common language runtime (CLR), a sizeable runtime that provides memory management, isolation by application domain, and a host of other application services. </td>
+</tr>
+<tr>
+<td>Includes CoreFx, a set of individual modular assemblies that you can add to your application as needed. Unlike the .NET Framework 4.x, which always makes the entire .NET Framework Class Library available, you can select only the assemblies that you need. For example, if you are developing a vector-based application, you can download the System.Numerics.Vectors package without the overhead of a large class library. This significantly reduces the size of your application and its dependencies.
+</td>
+<td>Includes the .NET Framework Class Library, a large, monolithic library with thousands of classes and many thousands of members. Whether or not your application uses individual types and their members (and most applications take advantage of only a small portion of the .NET Framework's functionality), they are always present and accessible. 
+</td>
+</tr>
+<tr>
+<td>Is suitable for a full range of modern applications, including applications that operate on small devices with constrained memory and storage.
+</td>
+<td>Is suitable for developing traditional Windows desktop applications, including Windows Forms (WinForms) and Windows Presentation Foundation (WPF) applications.
+</td>
+</tr>
+<tr>
+<td>Can be used to develop apps that take advantage of a number of technologies, such as ASP .NET Core, for building web applications; Windows Communication Foundation (WCF), for building services, including SOAP; and Workflow Foundation (WF), for building workflows. 
```

Comment:
Is this correct? I thought you couldn't build WCF services on .Net Core, only consume them.

From [dotnet/wcf](https://github.com/dotnet/wcf):

> WCF *service* applications should still be created with the full .NET Framework version.